server/controllers/article.js

```diff
@@ -17,17 +15,25 @@ const createArticle = async (req, res) => {
       categoryId
     });
 
-    res.status(201).json({
+    return res.status(201).json({
       status: 'success',
       message: 'Article created successfully',
       data: newArticle
     });
   } catch (error) {
-    res.status(500).json({
+    return res.status(500).json({
       status: 'failed',
       message: 'Server error'
```

Comment:
Instead of `server error`, could it be `error.message` instead for the user to know what went wrong?

Reply:
Ok